cmd/command__id/main.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"fmt"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	trivy_checks "github.com/aquasecurity/trivy-checks"
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	commandPath = "commands/kubernetes"
+)
+
+func main() {
+	ids, err := GetCommandIDRange()
+	if err != nil {
+		panic(err)
+	}
+	for i := 0; i < len(ids); i++ {
+		if !ids[i] {
+			println(fmt.Sprintf("%s-%04d", "CMD", i+1))
+		}
```

Comment:
I think it's enough to print the next available one. Or else the output is too verbose.

```shell
<snip>
CMD-9985
CMD-9986
CMD-9987
CMD-9988
CMD-9989
CMD-9990
CMD-9991
CMD-9992
CMD-9993
CMD-9994
CMD-9995
CMD-9996
CMD-9997
CMD-9998
CMD-9999
```


See below for how we do it for checks:
```shell
$ make id

The following IDs are free - choose the one for the service you are targeting.

             AWS: AVD-AWS-0344
             AZU: AVD-AZU-0052
          CLDSTK: AVD-CLDSTK-0002
             DIG: AVD-DIG-0010
              DS: AVD-DS-0031
             GCP: AVD-GCP-0069
             GIT: AVD-GIT-0005
             KCV: AVD-KCV-0093
             KSV: AVD-KSV-0123
            KUBE: AVD-KUBE-0003
             NIF: AVD-NIF-0022
             OCI: AVD-OCI-0002
          OPNSTK: AVD-OPNSTK-0006

```